services/secrets/src/api.js

```diff
@@ -101,6 +101,7 @@ builder.declare({
   method: 'get',
   route: '/secrets',
   name: 'list',
+  scopes: 'secrets:list-secrets',
```

Comment:
I thought this should be `service:secrets:list-secrets`?

Reply:
See https://github.com/taskcluster/taskcluster/pull/3615#discussion_r499785380